crates/solvers/src/domain/solver/baseline.rs

```diff
@@ -49,8 +50,8 @@ impl Baseline {
 
         auction
             .orders
-            .iter()
-            .take_while(|_| auction.deadline.remaining().is_some())
+            .par_iter()
+            .take_any_while(|_| auction.deadline.remaining().is_some())
```

Comment:
Bit unfortunate that we don't really take advantage of prioritized orders anymore but I guess `Baseline` is fast enough to go through all orders so it wouldn't really matter anyway.

Reply:
Hmm... Good point - maybe it makes sense for the baseline solver to keep it as it was. Also it would allow for better concurrency.

Reply:
Maybe we should use `tokio::task::spawn_blocking` for this.

Reply:
Will leave this for another time, its a larger change.

Reply:
https://github.com/cowprotocol/services/pull/1852